webapp/app.py

```diff
@@ -170,6 +173,13 @@ def utility_processor():
 app.add_url_rule(
     "/security/<regex('(cve-|CVE-)\\d{4}-\\d{4,7}'):cve_id>", view_func=cve
 )
+app.add_url_rule("/security/cve", view_func=create_cve, methods=["POST"])
```

Comment:
You were also talking of supporting adding multiple CVEs here. Did you decide against that? Or were you gonna do it in a follow-up PR instead?

Reply:
Well until I don't have a script that sends me a bulk of CVEs (or a list of CVEs) I cannot test that code, so it will not happen in this PR. Or if you prefer I can separate them.